<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="messageEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>545</y>
      <width>991</width>
      <height>91</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type your message...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1074</width>
      <height>471</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>544</y>
      <width>84</width>
      <height>93</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>320</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.98</string>
    </property>
    <property name="placeholderText">
     <string>Server ip adress...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>4</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>STATUS:OFFLINE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
